mlflow/store/sqlalchemy_store.py

```diff
@@ -10,6 +10,7 @@
 from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, RESOURCE_ALREADY_EXISTS, \
     INVALID_STATE, RESOURCE_DOES_NOT_EXIST
 from mlflow.utils.mlflow_tags import MLFLOW_PARENT_RUN_ID, MLFLOW_RUN_NAME
+from mlflow.utils.search_utils import does_run_match_clause
```

Comment:
Btw, a couple things I noticed while trying to get this to work in the UI:

- source_type and status should remain integers in order for the conversion to-and-from MLflow entities to work (as opposed to being stored as the string representation).
- We will need to include some notion of an artifact_directory within sqlalchemy_store in order to propagate this to newly-created experiments that do not specify an artifact_location.

Besides these two, the basics of MLflow and UI (including search) do seem to work!

Reply:
Thanks! We certainly need to address for an end-to-end working SQL store. I can handle those in a following PR.